pr_check.sh

```diff
@@ -28,4 +28,28 @@ export KUBEBUILDER_ASSETS=$PWD/kubebuilder_2.3.1_linux_amd64/bin
 IMG=$IMAGE_NAME:$IMAGE_TAG make docker-build
 IMG=$IMAGE_NAME:$IMAGE_TAG make docker-push
 
-docker run -i -v $PWD:/srcroot -e IMAGE_NAME=$IMAGE_NAME -e IMAGE_TAG=$IMAGE_TAG -e QUAY_USER=$QUAY_USER -e QUAY_TOKEN=$QUAY_TOKEN -e MINIKUBE_HOST=$MINIKUBE_HOST -e MINIKUBE_ROOTDIR=$MINIKUBE_ROOTDIR -e MINIKUBE_USER=$MINIKUBE_USER quay.io/psav/clowder_pr_check:blarg24 /srcroot/build/pr_check_inner.sh
+CONTAINER_NAME="clowder-pr-check-$ghprbPullId"
```

Comment:
`ghprbPullId` is a Jenkins var for the PR number, correct? So this would be `clowder-pr-check-214` ?

Reply:
Correct- they use the github pull request builder which provides this env var.

If you were to run this script locally, your container name would just end up being `clowder-pr-check-` which isn't great, but not terrible either.

Reply:
Okay gotcha. This was mostly for my education on what that was doing. Ty!